src/sql/parts/grid/views/editData/editData.component.ts

```diff
@@ -65,6 +72,10 @@ export class EditDataComponent extends GridParentComponent implements OnInit, On
 	public overrideCellFn: (rowNumber, columnId, value?, data?) => string;
 	public loadDataFunction: (offset: number, count: number) => Promise<IGridDataRow[]>;
 
+	@ViewChildren('slickgrid') slickgrids: QueryList<SlickGrid>;
```

Comment:
Does this need to be a list?  When would there ever be more than 1 slickgrid in the edit data scenario?